tests/static/itemset/test1.xml

```diff
@@ -23,7 +23,7 @@
 		</model>
 	</h:head>
 	<h:body>
-		<select1 ref="/data/question1">
+		<select1 ref="/data/question1" appearance="minimal">
```

Comment:
This did test it, but I suppose it was a little too sneaky. I added a more explicit test.